README.md

```diff
@@ -65,4 +92,205 @@
     - [Expo Documentation](https://docs.expo.dev/)
     - [Golang Documentation](https://go.dev/doc/)
    
-   
+## License
```

Comment:
probably best to put in a separate `LICENSE` or `COPYING` file instead of the `README.md` (if you want a mention of Apache in the README that's probably fine), I'm not sure if GitHub will auto detect the license if it's not (and it's a lot of stuff in the README that most people won't read)

Also, just curious, why Apache? Most of the other OSC projects seem to be MIT, but some are AGPL or GPL. I don't have a strong preference either way but was just wondering